scripts/afs_remote_fix.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+"""
+afs-remote-fix is a script for repointing your afs git remotes towards github.
+
+It will check if you're currently in a git ioc repo with an afs remote as your origin,
+switching it to your fork and adding an upstream remote to pcdshub.
+
+An afs remote is a git remote with a path that looks something like this:
+/afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/common/ims.git
+
+If this path is your origin, after running the script you will instead have:
+
+origin          git@github.com:your-username/ioc-common-ims.git
+upstream        git@github.com:pcdshub/ioc-common-ims.git
+
+If the origin is not an afs path, this script will exit without taking any action.
+
+The script will prompt you for your github username, which is not necessarily
+the same as your slac username.
+"""
+
+import subprocess
+
+
+def main() -> int:
+    origin = subprocess.check_output(
+        ["git", "remote", "get-url", "origin"], universal_newlines=True
+    ).strip()
+    if not origin.startswith("/afs/"):
+        print(f"Your origin {origin} is not an afs repo.")
+        return 1
+    elif "/ioc/" not in origin:
+        print(f"Your origin {origin} is not an ioc repo.")
+        return 1
+    _, ioc_path = origin.split("/ioc/")
+    repo_name = f"ioc-{ioc_path.replace('/', '-', 1)}"
+    username = input("Please input your github username:\n")
+    new_origin = f"git@github.com:{username}/{repo_name}"
+    new_upstream = f"git@github.com:pcdshub/{repo_name}"
+    print("\nPlanning to set:")
+    print(f"origin to {new_origin}")
+    print(f"upstream to {new_upstream}")
+    confirm = input("Confirm? y/n\n")
+    if not confirm.lower().startswith("y"):
+        return 1
+    subprocess.run(["git", "remote", "set-url", "origin", new_origin])
```

Comment:
Are we certain everyone has a remote called origin?  Maybe the more consistent thing to do is remove all the remotes then create `origin` and `upstream`

Reply:
You're right, this is a big assumption. This is definitely what you get if you just `git clone` from afs. Let me check what happens when you use `epics-checkout`.

Reply:
Here's the wild structure you get from `epics-checkout`:
```
zlentz@psbuild-rhel7-03:~/.../ims/ims-git(master +)$ pwd
/cds/home/z/zlentz/temp/afs_ioc_checks/ims/ims-git
zlentz@psbuild-rhel7-03:~/.../ims/ims-git(master +)$ git remote -v
origin  /afs/slac/g/cd/swe/git/repos/package/epics/ioc/common/ims.git (fetch)
origin  /afs/slac/g/cd/swe/git/repos/package/epics/ioc/common/ims.git (push)
```

Reply:
I guess if someone was savvy enough to change their remote names from the default, they're probably savvy enough to fix this without the help of the script?

Reply:
That's my current thought too, but I'm biased by not wanting to do more work on this. I'm happy to change the behavior if it would be helpful.

Reply:
I think this is good enough for the purpose.  probably more important to get this out there for the migration